README.md

```diff
@@ -1,33 +1,53 @@
-# Project
+# Azure Container Apps for Visual Studio Code
 
-> This repo has been populated by an initial template to help get you started. Please
-> make sure to update the content to build a great experience for community-building.
+<!-- region exclude-from-marketplace -->
 
-As the maintainer of this project, please make a few updates:
+[![Version](https://vsmarketplacebadge.apphb.com/version/ms-azuretools.vscode-azurecontainerapps.svg)](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-azurecontainerapps) [![Installs](https://vsmarketplacebadge.apphb.com/installs-short/ms-azuretools.vscode-azurecontainerapps.svg)](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-azurecontainerapps) [![Build Status](https://dev.azure.com/ms-azuretools/AzCode/_apis/build/status/vscode-azurecontainerapps?branchName=main)](https://dev.azure.com/ms-azuretools/AzCode/_build?definitionId=39&branchName=main)
```

Comment:
The build status shows as not set up yet. Will that populate once this PR merges or is there a typo in the URL?

Reply:
I believe that the url is correct.  When I click on it, it goes to the right branch/build.  Maybe it doesn't work from VS Code's markdown preview?